### DISCUSSION TITLE
Why speach data for punctuation and capitalization task?

### DISCUSSION THREAD

**Discussion by FaresAbanmy** [NVIDIA-NeMo/NeMo#2868] [Q&A] [answered]
As mentioned [here](https://ngc.nvidia.com/catalog/models/nvidia:nemo:punctuation_en_bert) that speach data has been used for the punctuation and capitalization model.

Do you use the speach data on training or testing?

If training, how? and why?
If testing, how can you evaluate it?

Thanks

**Comment by VahidooX** [comment]
@ekmb please take a look at this question.

**Comment by ekmb** [comment, marked as answer]
We used only text data from the speech datasets listed, no audio. The datasets were chosen to add variety to the training data. Gutenberg books contain many long complex sentences (this improves performance on commas), while the Fisher corpus contains conversational data with many question marks. To evaluation the model, please use [this script](https://github.com/NVIDIA/NeMo/blob/main/examples/nlp/token_classification/punctuation_capitalization_evaluate.py).
